Gemfile.lock

```diff
@@ -51,8 +51,8 @@ GEM
       minitest (~> 5.1)
       thread_safe (~> 0.3, >= 0.3.4)
       tzinfo (~> 1.1)
-    addressable (2.8.1)
-      public_suffix (>= 2.0.2, < 6.0)
+    addressable (2.8.7)
```

Comment:
Just one more thing, a few gems were bumped here that I assume are unrelated (addressable, capybara, capybara-email, xpath). Could we move those changes to #298 (if needed for Rails 5)?